packages/zoltan2/src/algorithms/partition/Zoltan2_AlgMultiJagged.hpp

```diff
@@ -7000,8 +7001,7 @@ void Zoltan2_AlgMJ<Adapter>::partition(
         distributor.doPostsAndWaits<mj_gno_t>(sent_gnos, 1, received_gnos());
       }
       {
-
-        ArrayView<mj_part_t> sent_partnos(partId.getRawPtr(), result_num_local_coords);
+        ArrayView<mj_part_t> sent_partnos(partId());
```

Comment:
This change looks fine.  I don't understand why it's needed, but it doesn't hurt anything.  Do you know if the failing test might have had no data on some process (so that getRawPtr would be NULL)?

Also, I'd be curious whether eliminating the temporary sent_partnos would be OK, just passing partID() to distributor.doPostsAndWaits.

I see the code is already merged; that's fine.

Reply:
I merged in hopes of fixing #2531.  It didn't help.

Reply:
@kddevin Yes, some processes have no data, so getRawPtr() would be null.